Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponent_Source.jinja

```diff
@@ -1779,9 +1785,9 @@ namespace {{ Component.attrib['Namespace'] }}
     {
         m_netBindComponent = netBindComponent;
         {{ DefineNetworkPropertyEditConstruction(Component, 'Authority', 'Server', ComponentBaseName)|indent(8) -}}
-{{ DefineNetworkPropertyEditConstruction(Component, 'Authority', 'Client', ComponentBaseName)|indent(8) -}}
-{{ DefineNetworkPropertyEditConstruction(Component, 'Authority', 'Autonomous', ComponentBaseName)|indent(8) -}}
-{{ DefineNetworkPropertyEditConstruction(Component, 'Autonomous', 'Authority', ComponentBaseName)|indent(8) }}
+        {{ DefineNetworkPropertyEditConstruction(Component, 'Authority', 'Client', ComponentBaseName)|indent(8) -}}
```

Comment:
Be careful with these whitespace changes.. you really need to validate each and every case generates the expected formatting. I went through and undid a bunch of these indentation changes recently because the generated files were all wrong.